float32[] wheel_spd
